.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Issues with the Steam runtime
+
+---
+
+#### Your system information
+
+* Steam Runtime Version: 
+* Distribution (e.g. Ubuntu): 
+* Link to your full system information in a [Gist](https://gist.github.com/): 
+  <!-- Please wait for the extended system infomation to be collected by Steam -->
+* Have you checked for system updates?: [Yes/No]
+
```

Comment:
```suggestion
* Are you using the experimental SteamLinuxRuntime compatibility tool?: [Yes/No]

```

Reply:
I adjusted this a little bit when adding it to the second iteration. Does that work for you?